.github/workflows/build.yaml

```diff
@@ -70,6 +70,7 @@ jobs:
 
   pre-build-ui:
     strategy:
+      fail-fast: false
```

Comment:
Is it a leftover from testing or why fail-fast `pre-build-ui` but not have the same for `pre-build-go-binaries`?

Reply:
> Is it a leftover from testing or why fail-fast `pre-build-ui` but not have the same for `pre-build-go-binaries`?

It is needed for `pre-build-ui` to get RHACS_BRANDING if STACKROX_BRANDING fails. 
It is not needed in `pre-build-go-binaries` because currently if a single arch fails the multi arch fails. I've a follow on issue to see if we want to change that behavior.  https://issues.redhat.com/browse/ROX-24862